source/templates.tex

```diff
@@ -7568,7 +7568,7 @@
 \item
 the specified member is not a template where a template is required, or
 \item
-the specified member is not a non-type where a non-type is required.
+the specified member is not a constant where a constant is required.
```

Comment:
The sentence applies to the following (which EDG and MSVC seems to get wrong: https://godbolt.org/z/j1hxG4f8e):
```cpp
template <typename T>
auto *f() {
  static constexpr auto x = sizeof (T::M); // sizeof _unary-expression_
  return &x;
}
struct Q { using M = int; };
void g() {
  f<Q>();
}
```

```suggestion
the specified member is not a non-type, non-template where a non-type, non-template is required.
```

Reply:
Good catch, I completely missed that